addons/mail/static/src/css/mail.css

```diff
@@ -1104,4 +1103,39 @@
     .openerp .oe_sidebar_suggestion .oe_suggest_title:hover .oe_suggestion_remove,
     .openerp .oe_sidebar_suggestion .oe_suggest_items:hover .oe_suggestion_remove_item {
         visibility: visible;
+    }
+/* ----------------------- @ Mention functionality ----------------------- */
```

Comment:
add your new css class in a new file called `mail_mention.css`. Don't use the `o_timeline_` prefix but `o_mail_mention_` since it is not timeline specific.